pyomo/core/expr/expr_pyomo5.py

```diff
@@ -1178,6 +1178,13 @@ def evaluate_expression(exp, exception=True, constant=False):
             raise
         return None
 
+    except TypeError:
```

Comment:
Snuck this one into another PR, eh? :)

Reply:
Not really.  Accidentally committed it here when I thought I was on another branch, but had already pushed it up by the time I caught the mistake.